### PR TITLE
Docker Install Update

### PR DESCRIPTION
**Pull request by R8s6**
#### Summary

Changing default set up uid:gid = 2000:2000 to 1000:1000 and explaining how to find out and use the actual uid:gid.

#### Ticket Link

Fixes https://github.com/mattermost/docs/issues/3883

### CODE REVIEW

File: source/install/prod-docker.rst [hunk 1]
```diff
@@ -32,7 +32,17 @@ Docker Setup on Ubuntu
       sudo curl -L "https://github.com/docker/compose/releases/download/$dockerComposeVersion/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
       sudo chmod +x /usr/local/bin/docker-compose
 
-3. **Deploy the Mattermost Docker** setup by running:
+3. **Deploy the Mattermost Docker** 
+
+Find out the :code:`uid` (user ID) and :code:`gid` (group ID) of the user to run the docker, by running:
```
Comment (justinegeffen) [inline source/install/prod-docker.rst:37]:
```suggestion
You can get the :code:`uid` (user ID) and :code:`gid` (group ID) of the Docker user by running:
```

File: source/install/prod-docker.rst [hunk 2]
```diff
@@ -32,7 +32,17 @@ Docker Setup on Ubuntu
...
+
+   .. code:: bash
+   
+       id <username>
+
+Replace :code:`<username>` with the actual username. The following setup assumes the result above is :code:`uid=1000` and :code:`gid=1000`.
+
+Setup by running:
```
Comment (justinegeffen) [inline source/install/prod-docker.rst:45]:
```suggestion
Set up using:
```

**Review by svelle — APPROVED** [review]
LGTM

**Comment by justinegeffen** [comment]
Hi @R8s6! This is looking great. I've made some minor suggestions. If you're open to reviewing them that'd be awesome. :) Thank you!

**Comment by R8s6** [comment]
> Hi @R8s6! This is looking great. I've made some minor suggestions. If you're open to reviewing them that'd be awesome. :) Then once you've signed the CLA, we can merge this PR. :) Thank you!

Hi @justinegeffen , thanks for the updates, they look perfect.

It seems that CLA has been signed, if not, please let me know.

Also please let me know if anything else is required from me, thank you very much!

**Review by justinegeffen — APPROVED** [review]

**Comment by justinegeffen** [comment]
Thank you, @R8s6! Awesome changes, much appreciated. :)

**Pull request merged**